ScriptureRenderingPipeline/Renderers/BibleRenderer.cs

```diff
@@ -16,152 +16,152 @@
+        Directory.CreateDirectory(Path.Join(destinationDir, abbreviation));
+        var outputBook = new OutputBook()
+        {
+          Slug = abbreviation,
+          Label = title
+        };
+        foreach (var chapter in chapters)
+        {
+          var tmp = new USFMDocument();
+          tmp.Insert(chapter);
+          outputTasks.Add(File.WriteAllTextAsync(Path.Join(destinationDir, abbreviation, $"{chapter.Number.ToString()}.html"), renderer.Render(tmp)));
+          outputBook.Chapters.Add(new OutputChapters()
+          {
+            Number = chapter.Number.ToString(),
+            Label = chapter.PublishedChapterMarker
+          });
+        }
+        index.Bible.Add(outputBook);
 
-                // Since the print all page isn't going to broken up then just write stuff out here
-                printBuilder.AppendLine(content);
-            }
+        // Since the print all page isn't going to broken up then just write stuff out here
+        printBuilder.AppendLine(content);
+      }
 
-            // If we have something then create the print_all.html page and the index.html page
-            if (documents.Count > 0)
-            {
-                outputTasks.Add(File.WriteAllTextAsync(Path.Join(destinationDir, "print_all.html"), printTemplate.Render(Hash.FromAnonymousObject(new { content = printBuilder.ToString(), heading }))));
-            }
-            outputTasks.Add(File.WriteAllTextAsync(Path.Join(destinationDir, "index.json"), JsonSerializer.Serialize(index)));
+      // If we have something then create the print_all.html page and the index.html page
+      if (documents.Count > 0)
+      {
+        outputTasks.Add(File.WriteAllTextAsync(Path.Join(destinationDir, "print_all.html"), printTemplate.Render(Hash.FromAnonymousObject(new { content = printBuilder.ToString(), heading }))));
+      }
+      outputTasks.Add(File.WriteAllTextAsync(Path.Join(destinationDir, "index.json"), JsonSerializer.Serialize(index)));
 
-            await Task.WhenAll(outputTasks);
+      await Task.WhenAll(outputTasks);
+    }
+    /// <summary>
+    /// Load all USFM files in a directory inside of the ZipFileSystem
+    /// </summary>
+    /// <param name="directory">A ZipFileSystem to load from</param>
+    /// <returns>A list of USFM files</returns>
+    static async Task<List<USFMDocument>> LoadDirectoryAsync(ZipFileSystem directory)
+    {
+      USFMParser parser = new USFMParser(new List<string> { "s5" }, true);
+      var output = new List<USFMDocument>();
+      foreach (var f in directory.GetAllFiles(".usfm"))
+      {
+        var tmp = parser.ParseFromString(await directory.ReadAllTextAsync(f));
+        // If we don't have an abbreviation then try to figure it out from the file name
+        var tableOfContentsMarkers = tmp.GetChildMarkers<TOC3Marker>();
+        if (tableOfContentsMarkers.Count == 0)
+        {
+          var bookAbbrivation = GetBookAbberviationFromFileName(f);

```

Comment:
Typo: Should be "bookAbbreviation" -- this typo appears in a lot of places in this file.